bpf/lib/icmp6.h

```diff
@@ -398,7 +398,7 @@ icmp6_host_handle(struct __ctx_buff *ctx __maybe_unused)
 
 	type = icmp6_load_type(ctx, ETH_HLEN);
 	if (type == ICMP6_NS_MSG_TYPE)
-		return icmp6_handle_ns(ctx, ETH_HLEN, METRIC_INGRESS);
+		return CTX_ACT_OK;
```

Comment:
This should move under the `#ifdef ENABLE_HOST_FIREWALL` guard below, I think? If we're not using the host firewall, we'll return `CTX_ACT_OK` in any case.

Given the function will become a simple `return CTX_ACT_OK`, we could even gate the call to icmp6_host_handle() and result check behind `ENABLE_HOST_FIREWALL` in the callers?

Reply:
Good point, thanks!